src/theme/ColorModeToggle/index.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import clsx from 'clsx';
+import useIsBrowser from '@docusaurus/useIsBrowser';
+import { translate } from '@docusaurus/Translate';
+import IconLightMode from '@theme/Icon/LightMode';
+import IconDarkMode from '@theme/Icon/DarkMode';
+import styles from './styles.module.css';
+function ColorModeToggle({ className, value, onChange }) {
+  const isBrowser = useIsBrowser();
+  const title = translate(
+    {
+      message: 'Switch between dark and light mode (currently {mode})',
```

Comment:
> @VitroidFPV this seems not to be working the first time I enter in the page. When I enter in the page, it thinks it is at dark mode but is in light mode: ![image](https://user-images.githubusercontent.com/2673520/257743249-e0d0aa74-f8ad-4a78-b4b3-74008e344018.png)
> 
> Maybe can we add an "auto" mode too, that uses the one selected by default in the SO?
> 
> I love the light mode, thanks for it!!

This is an hydration mismatch caused by React issue, seems that it will fixed in react18, else im working on a solution to read init colorMode state